packages/account/src/Components/poi/status/unsupported/__test__/unsupported.spec.js

```diff
@@ -1,12 +0,0 @@
-import React from 'react';
```

Comment:
This test case is outdated and invalid since the component has been heavily modified, so we will create new test cases separately